imagebuildah/stage_executor.go

```diff
@@ -950,7 +957,14 @@ func (s *StageExecutor) Execute(ctx context.Context, base string) (imgID string,
 			// Check if there's already an image based on our parent that
 			// has the same change that we just made.
 			if checkForLayers {
-				cacheID, err = s.intermediateImageExists(ctx, node, addedContentSummary, s.stepRequiresLayer(step))
+				err = s.executor.localLayerProvider.PopulateLayer(ctx, s.builder.TopLayer)
+				if err != nil {
+					return "", nil, errors.Wrap(err, "error checking if cached image exists from a previous build")
```

Comment:
Please do not start error messages with "error"

Reply:
resolved